src/archanaNJun20/ArrayIntersection.java

```diff
@@ -0,0 +1,43 @@
+package archanaNJun20;
+
+public class ArrayIntersection {
+	double[] getIntersectionOfArray(double[] array1, double[] array2){		
+		int  arraycount=0;
+		int length=0;
+		if(array1.length>array2.length)
+			length= array1.length;
```

Comment:
For intersection of an array, you should create array length of min array. You have written logic for max array. Change it.

Reply:
The code is updated.
Thank you :)